docs/Load-Sample-Cancer-Study.md

```diff
@@ -1,50 +1,52 @@
 
-    CONNECTOR_JAR=$(echo $CATALINA_HOME/lib/mysql-connector-java-*-bin.jar)
+## Validating the Sample Study
 
-Make sure the ``CONNECTOR_JAR`` variable is pointing to one jar:
+First, validate `study_es_0`.
 
-    echo $CONNECTOR_JAR
-    # output e.g. /Library/Tomcat/lib/mysql-connector-java-5.1.35-bin.jar
+To do so, go to the importer folder: 
 
-Second, set the path to the core jar of cbioportal:
+```
+cd <your_cbioportal_dir>/core/src/main/scripts/importer
+```
 
-    CORE_JAR=$(echo $PORTAL_HOME/core/target/core-*.jar)
+and then run the following command:
 
-Again, make sure the ``CORE_JAR`` variable is pointing to one jar:
+```
+./validateData.py -s ../../../test/scripts/test_data/study_es_0/ -n
+```
 
-    echo $CORE_JAR
-    # output e.g. /Users/inodb/git/cbioportal/core/target/core-0.1.2-SNAPSHOT.jar   
+If all goes well, you should see the final output message:
 
-Then, load the meta-data for the sample study:
+```
+Validation of study succeeded with warnings.
+```
 
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-study --meta-filename portal-study/meta_study.txt
+## Importing the Sample Study
 
-Then, load copy number, mutation data and expression data:
+To import the sample study:
 
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-study-data --meta-filename portal-study/meta_CNA.txt --data-filename portal-study/data_CNA.txt
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-study-data --meta-filename portal-study/meta_mutations_extended.txt --data-filename portal-study/data_mutations_extended.txt
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-study-data --meta-filename portal-study/meta_expression_median.txt --data-filename portal-study/data_expression_median.txt
+```
+cd <your_cbioportal_dir>/core/src/main/scripts/importer
+```
 
-Lastly, load the case sets and clinical attributes:
+and then run the following command:
 
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-case-list --meta-filename portal-study/case_lists
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-study-data --meta-filename portal-study/meta_clinical.txt --data-filename portal-study/data_clinical.txt
+```
+./metaImport.py -s ../../../test/scripts/test_data/study_es_0/ -n -o
```

Comment:
for this to run, you (today) will still need an environment variable called PORTAL_HOME pointing to the root of the source checkout.